sys/systems/summit.def

```diff
@@ -29,6 +29,7 @@ endif
 # Clang compiler
 # ifeq ($(CC), clang)
 #   C_COMPILER_MODULE = llvm/1.0-20190225
+#   C_COMPILER_MODULE = cuda/10.1.243; module use /sw/summit/modulefiles/ums/stf010/Core; module load llvm/12.0.0-latest
```

Comment:
Change to llvm/13.0.0-latest

Reply:
Changed it to llvm 13 and added some required flags.